### DISCUSSION TITLE
Sd Card

### DISCUSSION THREAD

**Discussion by Kmog2185** [rafaelvcaetano/melonDS-android#1158] [Ideas]
Even though it has been asked before, I do believe that SD Card support for Dsiware would be an important change due to the currently limited amount of games installable and how storage expansion would be game changing.

**Comment by christopher3242** [comment]
Good idea But pull requests like 4 GB 8 , 16 , 32 , 64 and 128 GB Sd card memory but too much I apology.  For @Kmog2185 

req @rafaelvcaetano
